### PR TITLE
Big-endian support for multiplayer

### PR DESCRIPTION
**Pull request by glebm**
Not sure if this works but it's a start -- needs testing.

The implementation can be improved in the future by moving byte-swapping into accessor methods, 
per https://github.com/diasurgical/devilutionX/pull/1416

Based on top of #5605

### CODE REVIEW

**Comment by AJenbo** [comment]
Previous PR for this (never got tested)

File: Source/items.cpp [hunk 1]
```diff
@@ -2697,7 +2697,11 @@ void CalcPlrItemVals(Player &player, bool loadgfx)
 		int8_t ticksPerFrame;
 		player.getAnimationFramesAndTicksPerFrame(graphic, numberOfFrames, ticksPerFrame);
 		LoadPlrGFX(player, graphic);
-		player.AnimInfo.changeAnimationData(player.AnimationData[static_cast<size_t>(graphic)].spritesForDirection(player._pdir), numberOfFrames, ticksPerFrame);
+		OptionalClxSpriteList sprites;
+		if (!HeadlessMode) {
+			sprites = player.AnimationData[static_cast<size_t>(graphic)].spritesForDirection(player._pdir);
+		}
+		player.AnimInfo.changeAnimationData(sprites, numberOfFrames, ticksPerFrame);
```
Comment (AJenbo) [inline Source/items.cpp:2704]: Oh nice I belive this is also what prevented me from testing a lot of the various game parts in the timedemo

**Comment by glebm** [comment]
Rebased

**Comment by AJenbo** [comment]
... merged the wrong one

**Pull request merged**